jac-uk/apply
# Displaying file upload types

## What's included?
Display the types of files that can be uploaded next to the file input

Closes #1026

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
See video:

https://github.com/jac-uk/apply/assets/115651787/26c0583f-482c-42ee-a6cc-f6f9f8f164b1

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
